src/node_internals.h

```diff
@@ -118,6 +118,9 @@ class NodeArrayBufferAllocator : public ArrayBufferAllocator {
  private:
   uint32_t zero_fill_field_ = 1;  // Boolean but exposed as uint32 to JS land.
   std::atomic<size_t> total_mem_usage_ {0};
+
+  std::unique_ptr<v8::ArrayBuffer::Allocator> allocator_{
```

Comment:
Would you mind adding a comment here about why this is being used instead of the regular Node.js allocation methods?

Reply:
Done; I kept it terse but happy to either expand it or discuss more here (or both!)